go-controller/pkg/ovn/ovn.go

```diff
@@ -279,8 +282,58 @@ func (oc *Controller) Run(wg *sync.WaitGroup) error {
 	oc.WatchNodes()
 
 	oc.WatchPods()
-	oc.WatchServices()
-	oc.WatchEndpoints()
+
+	// Services are handled differently depending on the Kubernetes API versions
+	// and the OVN configuration.
+	// We use a level triggered controller to handle services if k8s > 1.19,
+	// using endpoint slices instead endpoints if OVN is configured for dual stack.
+	if util.UseEndpointSlices(oc.client) && config.IPv4Mode && config.IPv6Mode {
```

Comment:
This guarantees no interference with the other way of handling services, when using slices only the controller handles LoadBalancers in OVN